src/Uno.UI/UI/Xaml/Documents/Hyperlink.wasm.cs

```diff
@@ -10,12 +11,16 @@ partial class Hyperlink
 		public Hyperlink() : base("a")
 		{
 			UpdateNavigationProperties(null, _defaultNavigationTarget);
-
+			Foreground = DefaultForeground;
 			PointerPressed += TextBlock.OnPointerPressed;
 			PointerReleased += TextBlock.OnPointerReleased;
 			PointerCaptureLost += TextBlock.OnPointerCaptureLost;
 		}
 
+		//UWP Hyperlink color
+		private static Brush GetDefaultForeground()
```

Comment:
Isn't this in a default style ?

Reply:
I don't think there's a style for Hyperlink since it doesn't inherit from Control